### PR TITLE
Preparing for 0.96.0.1

### PR DESCRIPTION
**Pull request by andreasabel**
Preparing for revision 0.96.0.0-r2 
- ghc 9.0 compat
- github Haskell CI

### CODE REVIEW

File: regex-posix.cabal [hunk 1]
```diff
@@ -23,23 +24,32 @@ extra-source-files:
...
+  GHC == 8.6.5
+  GHC == 8.8.4
+  GHC == 8.10.3
+  -- manually (AA, 2021-02-17):
+  GHC == 8.10.4
+  -- GHC == 9.0.1
 
 source-repository head
   type:     git
   location: https://github.com/hvr/regex-posix.git
```
Comment (phadej) [inline regex-posix.cabal:47]: this is worth updating too

File: regex-posix.cabal [hunk 2]
```diff
@@ -88,5 +98,4 @@ library
   if !impl(ghc >= 8)
       build-depends: fail == 4.9.*
 
-  ghc-options: -O2
```
Comment (phadej) [inline regex-posix.cabal:91]: Ditto removal of `-O2`. You cannot even to this in a revision IIRC.

File: regex-posix.cabal [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
 name:                   regex-posix
 version:                0.96.0.0
+x-revision:             2
 
 build-type:             Simple
 license:                BSD3
 license-file:           LICENSE
 copyright:              Copyright (c) 2007-2010, Christopher Kuklewicz
 author:                 Christopher Kuklewicz
 maintainer:             hvr@gnu.org
```
Comment (emilypi) [inline regex-posix.cabal]: could you bump the maintainer list here?

**Review by emilypi — COMMENTED** [review]
Seconding oleg's comments as well

File: regex-posix.cabal (hunk 2 above)
Comment (andreasabel) [reply regex-posix.cabal:91]: Ok, then I am going for releasing 0.96.0.1.

File: regex-posix.cabal (hunk 3 above)
Comment (andreasabel) [reply regex-posix.cabal]: Who should be on the list?

Comment (andreasabel) [reply regex-posix.cabal]: Clarified that with @hvr.

**Comment by andreasabel** [comment]
Following discussion with @hvr, I reverted back to just a revision, which is live on hackage.
Thanks for the reviews, @phadej and @emilypi !

**Pull request closed without merging**